lib/autoRest.ts

```diff
@@ -6,4 +9,90 @@ export interface AutoRest {
    * Create a new synchronous AutoRest generation process.
    */
   run(): void;
-}
\ No newline at end of file
+}
+
+/**
+ * A fake AutoRest facade implementation.
+ */
+export class FakeAutoRest implements AutoRest {
+  public run(): void {
+    throw new Error("Method not implemented.");
+  }
+}
+
+/**
+ * The parsed version of the swagger-to-sdk YAML block within an AutoRest readme.md file.
+ */
+export interface ReadmeMdSwaggerToSDKConfiguration {
+  /**
+   * The repositories specified.
+   */
+  repositories: RepositoryConfiguration[];
+}
+
+/**
+ * An individual repository configuration within an AutoRest readme.md swagger-to-sdk YAML block
+ * configuration.
+ */
+export interface RepositoryConfiguration {
+  /**
+   * The name of the GitHub repository this configuration applies to. If no organization is
+   * specified, then Azure will be used.
+   */
+  repo: string;
+  /**
+   * The list of commands that will be run (in order) after an SDK has been generated.
+   */
+  after_scripts: string[];
+}
+
+function findSwaggerToSDKYamlBlocks(parsedMarkdownNode: commonmark.Node | undefined | null): commonmark.Node[] {
```

Comment:
I didn't see that. This is a function that I threw together quickly. @fearthecowboy is supposed to extract the readme/YAML parser out of autorest sometime so that I can consume that directly.